src/neuroconv/tools/nwb_helpers/_configure_backend.py

```diff
@@ -12,9 +12,19 @@
 def configure_backend(
     nwbfile: NWBFile, backend_configuration: Union[HDF5BackendConfiguration, ZarrBackendConfiguration]
 ) -> None:
-    """Configure all datasets specified in the `backend_configuration` with their appropriate DataIO and options."""
+    """
+    Configure all datasets specified in the `backend_configuration` with their appropriate DataIO and options.
+
+    Parameters
+    ----------
+    nwbfile : pynwb.NWBFile
+        The in-memory pynwb.NWBFile object to configure.
+    backend_configuration : "hdf5" or "zarr", default: "hdf5"
```

Comment:
The typing here does not match the typehints

Reply:
Good eye

https://github.com/catalystneuro/neuroconv/pull/822